### DISCUSSION TITLE
Github Actions Workflow does not start (GHA default runner)

### DISCUSSION THREAD

**Discussion by Jack12816** [community/community#141884] [Actions]
### Select Topic Area

General

### Body

Triggered via release 51 minutes ago, Status: Queued.

And I cannot cancel the workflow, and no jobs start. Looks like an GHA malfunction.

**Comment by Sanidhyafeaturist** [comment]
if you can't cancel the workflow and no jobs have started, it may be a malfunction on GitHub's end. I recommend checking the GitHub Status page for any ongoing incidents and, if the issue persists, consider reaching out to GitHub Support for further assistance.
